src/styles/themes/_default-theme.scss

```diff
@@ -208,7 +208,25 @@
   --backdrop-filter: blur(5px);
 
   --anchor-color: var(--primary-color-70);
+  --anchor-color-disruptive: var(--disruptive-color-70);
```

Comment:
feel like you should invert the naming here so that sets of colors are grouped up

anchor-neutral-color
anchor-neutral-hover-color
anchor-neutral-visited-color